### PR TITLE
U fe 1

### PR DESCRIPTION
**Pull request by frossie**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -48,14 +48,150 @@
...
+   
+   This is a *descriptive* (not prescriptive) document describing details of current practice of a specific team in a specific area. For the actual developer guide, see developer.lsst.io
+
+SQuaRE uses Configuration as Code (aka “gitops”) practices for service deployment whenever practical.
+What this means is that by merging code or configuration to a special branch (typically master), it is *possible* to trigger an action (via an agent or supervisory system) that automatically results in the deployment of the new version of the service.
+
+Whether the new version of the service is *actually* deployed automatically (a practice known as continuous deployment) or whose deployment is gated behind a human decision making process is a matter of policy.
+The barrier to getting through that gate is itself a function of where a system lies on an axis of formality, with one end maximizing developer agency and the other maximizing organisational control.
+
+.. image:: _static/axis.png
```
Comment (rra) [inline index.rst]: The text in this image is unreadably small for me.

File: index.rst [hunk 2]
```diff
@@ -48,14 +48,150 @@
...
+
+-  The service’s maturity, including the number of users depending on the service
+   
+-  The service environment, including whether it’s production, staging or development environment
+   
+-  The service’s userbase, including whether it is a feature-seeking or risk averse population
+   
+-  The actual and reputational impact of disruptions to the service
+   
+-  The nature of the change being made.
```
Comment (rra) [inline index.rst]: Nit: This didn't render as a bullet list for some reason. Maybe it didn't like the blank lines between bullet points? I'm not sure.

File: index.rst [hunk 3]
```diff
@@ -48,14 +48,150 @@
...
+
+It’s easy to see that a major change in a mature high visibility service that many users depend on is on the opposite side of the axis that a cosmetic change in a service under development deployed on a sandbox.
+The thickier situations lie in between these extremes, and the aim of this document is to expose the heuristics by which we consider these questions.
```
Comment (rra) [inline index.rst]:
```suggestion
The trickier situations lie in between these extremes, and the aim of this document is to expose the heuristics by which we consider these questions.
```

File: index.rst [hunk 4]
```diff
@@ -48,14 +48,150 @@
...
+We also describe our extant processes around the update of Science Platform and Science Platform adjacent services, as these are likely to be of most interest to developers outside our team and since they exercise a lot of the decision space described above.
+
+Kubernetes Services and ArgoCD
```
Comment (rra) [inline index.rst]:
```suggestion
Kubernetes services and Argo CD
```

File: index.rst [hunk 5]
```diff
@@ -48,14 +48,150 @@
...
+
+It follows then that for gated deployments there are two aspects to getting a new version of a service deployed into production:
+
+1. Mechanics: Getting the right version of the service and its configuration merged into the deployment branch (typically master);
+2. Process: Going through the process that allows it to be synced into production.
+
+Mechanics
+---------
+
+[gafalefawr example]
```
Comment (rra) [inline index.rst]:
```suggestion
Assume that there is a new release of Gafaelfawr, the authentication service used by the Rubin Science Platform.
The following steps notify Argo CD that this upgrade is ready to be performed:

#. Update the Helm chart in the [charts](https://github.com/lsst-sqre/charts) repository under `charts/gafaelfawr`.
   Do this by updating the `appVersion` key in `Chart.yaml` and the `image.tag` key in `values.yaml` to point to the new release.
   Also update the version of the Helm chart in `Chart.yaml`.
   When this PR is merged, a new version of the chart will be automatically published via GitHub Actions.
#. Update the Argo CD configuration to deploy the new version of the chart.
   The `Chart.yaml` file in `services/gafaelfawr` in the [phalanx](https://github.com/lsst-sqre/phalanx) repository determines what version of the Gafaelfawr chart is installed by Argo CD.
   (Phalanx is the Argo CD configuration repository for the Rubin Science Platform.)
   Under the stanza for the `gafaelfawr` chart in the `dependencies` key, change the version to match the newly-released chart version.
   Once this PR is merged, Argo CD will notice there is a pending upgrade for Gafaelfawr.
```

**Review by rra — APPROVED** [review]
Looks good to me apart from the known problem with the table formatting.

File: index.rst (hunk 2 above)
Comment (frossie) [reply index.rst]: For reasons I cannot begin to comprehend, the problem was resolved when I added a section heading in the scope the bullet list was in 🤷‍♀️

File: index.rst (hunk 1 above)
Comment (frossie) [reply index.rst]: Fixed, as is the table formatting.

**Pull request merged**